docs/examples/writing-a-rule.md

```diff
@@ -0,0 +1,319 @@
+      "rule_id": "CUSTOM_0001",
+      "rule_message": "",
+      "rule_name": "long_description",
+      "rule_result": "FAIL",
+      "rule_severity": "Low",
+      "rule_summary": "IAM policies must have a description of at least 25 characters"
+    },
+    {
+      "controls": [
+        "CORPORATE-POLICY_1.1"
+      ],
+      "filepath": "../regula-ci-example/infra_tf/",
+      "platform": "terraform",
+      "provider": "aws",
+      "resource_id": "aws_iam_policy.basically_deny_all",
+      "resource_type": "aws_iam_policy",
+      "rule_description": "Per company policy, it is required for all IAM policies to have a description of at least 25 characters.",
+      "rule_id": "CUSTOM_0001",
+      "rule_message": "",
+      "rule_name": "long_description",
+      "rule_result": "PASS",
+      "rule_severity": "Low",
+      "rule_summary": "IAM policies must have a description of at least 25 characters"
+    }
+  ],
+  "summary": {
+    "filepaths": [
+      "../regula-ci-example/infra_tf/"
+    ],
+    "rule_results": {
+      "FAIL": 1,
+      "PASS": 1,
+      "WAIVED": 0
+    },
+    "severities": {
+      "Critical": 0,
+      "High": 0,
+      "Informational": 0,
+      "Low": 1,
+      "Medium": 0,
+      "Unknown": 0
+    }
+  }
+}
+```
+
+As you can see, the IAM policy `aws_iam_policy.basically_deny_all` passed the custom rule because its description is long enough, and `aws_iam_policy.basically_allow_all` failed because its description is too short. Check out the [Terraform itself](https://github.com/fugue/regula-ci-example/blob/master/infra_tf/main.tf) and you can confirm it.
+
+## CloudFormation example rule
+
+The process for writing custom rules to check CloudFormation is mostly the same as for Terraform -- the main difference is how you specify the resource type and attribute to check. Additionally, you need to specify `input_type := "cloudformation"` somewhere in the rule.
+
+For instance, this is how you'd write `long_description` as a CloudFormation rule. Note that `resource_type` and `Description` are different from the Terraform rule, and `input_type` is present:
+
+```
+package rules.long_description_cfn
+
+__rego__metadoc__ := {
+  "id": "CUSTOM_0001",
+  "title": "IAM policies must have a description of at least 25 characters",
+  "description": "Per company policy, it is required for all IAM policies to have a description of at least 25 characters.",
+  "custom": {
+    "controls": {
+      "CORPORATE-POLICY": [
+        "CORPORATE-POLICY_1.1"
+      ]
+    },
+    "severity": "Low"
+  }
+}
+
+input_type := "cloudformation"
+
+resource_type = "AWS::IAM::ManagedPolicy"
+
+default allow = false
+
+allow {
+  count(input.Description) >= 25
+}
+```
+
+## What's next?
+
+Congratulations, you wrote a simple custom rule! :tada: To learn more about writing rules, check out [Writing Rules](../development/writing-rules.md). Or, continue onward to learn about [integrations](regula-conftest.md).
```

Comment:
Could be very specific to me, but the `continue onward` wording confused me for a sec here since the next button at the bottom takes you to the `Contributing` page.

Reply:
Oops! I forgot I changed around the navigation after writing this. Fixed, thanks!